mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_backup_vault_test.go

```diff
@@ -0,0 +1,51 @@
+package backupdr_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"testing"
+)
+
+func TestAccDataSourceGoogleBackupDRBackupVault_basic(t *testing.T) {
+	t.Parallel()
+
+	random_suffix := acctest.RandString(t, 10)
+	context := map[string]interface{}{
+		"random_suffix": random_suffix,
+	}
+	id := "bv-" + random_suffix
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckBackupDRBackupVaultDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleBackupDRBackupVault_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.google_backup_dr_backup_vault.fetch-bv", "backup_vault_id", id),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleBackupDRBackupVault_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_backup_dr_backup_vault" "test-bv" {
+  location = "us-central1"
+  backup_vault_id = "bv-%{random_suffix}"
```

Comment:
```suggestion
  backup_vault_id = "tf-test-bv-%{random_suffix}"
```
as we also sweepers that clean up test resources with `tf-test` prefix